packages/react-form-renderer/src/hooks/use-field-api.js

```diff
@@ -32,11 +34,20 @@ const useFieldApi = ({ name, initializeOnMount, component, render, ...props }) =
    */
   const fieldClearedValue = Object.prototype.hasOwnProperty.call(props, 'clearedValue') ? props.clearedValue : formOptions.clearedValue;
 
+  let overrideProps = {};
+  if (props.actions) {
+    Object.keys(props.actions).forEach((prop) => {
```

Comment:
There is a question of `undefined` values here. What if the actions is this:
```jsx
{
  someAction: undefined
}
```

Do we want to override the existing action with the `undefined` value and possibly risk runtime error or do we want to preserve the original prop?

Maybe we can make it more specific and if the user want to override the prop with an empty one, we can enforce ussage of `false` or `null` or something similar?

Reply:
What about to enforce this in the schema validator? No undefined values, etc.

Reply:
Well, the question is more about if we should do it or not. I can imagine if somebody is generating this, thew will have `undefined` keys instead of deleting them. Maybe use the `false`/`null` to override the prop to be empty?

Reply:
But why, if you want to override props in the schema, just don't write the original ones... :D  and if someone generates this, so they will just write their generator without generating falsy values.

This should serve to map actions to props, not to override them...

Reply:
Yup that is true.